src/fund_my_watcard/config_file.py

```diff
@@ -16,7 +16,8 @@
     DECRYPTING_CONFIG_FILE_FAILED,
 )
 
-CONFIG_FILE_PATH = os.path.expanduser("~") + "/.watcard_config"
+WATCARD_PATH = os.path.expanduser("~")
```

Comment:
I would suggest to call it `USER_DIR` or `HOME`. What do you think?

Reply:
Sure. `USER_DIR` is better.